### ISSUE TITLE
Light / Dark Theme Variable Names

### ISSUE THREAD

**Issue by TerminalFi** [meetio-theme/merge-meetio-theme#3]
The following variables need to be renamed, so that they make sense in both light and dark themes.

They make sense in the Dark theme but not the Light theme.

```
accentLight
accentLighter
accentLightest
accentDark
accentDarker
accentDarkest
```

**Suggestion**

```
accent1
accent2
accent3
accent4
accent5
accent6
```

**Comment by mauroreisvieira** [comment]
Hi @TheSecEng.
Go ahead, I don't know if using `accent1` instead of `accentLight` it's clear enough four us, when we are using, but let's try.

**Comment by TerminalFi** [comment]
> Hi @TheSecEng.
> Go ahead, I don't know if using `accent1` instead of `accentLight` it's clear enough four us, when we are using, but let's try.

I agree, it isn't ideal. I am open to a better implementation. Maybe we can somehow get rid of the accent variables in favor of something else.

**Comment by mauroreisvieira** [comment]
Yes, we need to understand well where we are using this kind of variables, to have a better understanding of usages, its help us to understand what name can we use.

**Comment by jhosseeth** [comment]
Hi, I tried to change to darkness theme but nothing is happening

> // Settings in here override those in "Default/Preferences.sublime-settings",
// and are overridden in turn by syntax-specific settings.
{
	"theme": "Merge Darker.sublime-theme",
    "color_scheme": "Meetio Darker.sublime-color-scheme",
}

**Comment by TerminalFi** [comment]
Did you clone this repo down into `Sublime Merge/Packages`

**Comment by ZeroCommits** [comment]
@TerminalFi do I need a license to make this work?
